subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -65,6 +68,13 @@
   int[] show_stack_pointers = new int[7];
   String[] show_stack_data = new String[6];
 
+  String[] show_power_expected = {
+    "Interface", "Admin", "Pri", "Oper", "Power", "Device", "Class", "Max", "\n"
+  };
+
+  int[] show_power_pointers = new int[9];
+  String[] show_power_data = new String[8];
```

Comment:
is this length-1, then?  Much better to use any kind of named calculation than it is straight-up constants.

Reply:
implemented length where applicable